.CI/build_all

```diff
@@ -14,11 +14,16 @@ if [ -z "$GH_TOKEN" ]; then
     # Install and configure conda environment.
     curl -L -O https://raw.githubusercontent.com/pelson/Obvious-CI/master/bootstrap-obvious-ci-and-miniconda.py
     python bootstrap-obvious-ci-and-miniconda.py ~/miniconda x64 3 --without-obvci && source ~/miniconda/bin/activate root
+    mkdir -p ~/miniconda/conda-meta
+    echo 'conda-build 1.21.7' > ~/miniconda/conda-meta/pinned
+    echo 'setuptools 23.0.0' >> ~/miniconda/conda-meta/pinned
```

Comment:
So you found issues with `setuptools` too? downgrading conda-build was not enough to fix it?
Instead of adding this here can you pin `setuptools` in the recipe? See below.